bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/editor/SummernoteEditorOverlayCssReference.java

```diff
@@ -0,0 +1,33 @@
+package de.agilecoders.wicket.extensions.markup.html.bootstrap.editor;
+
+import org.apache.wicket.request.resource.CssResourceReference;
+
+/**
+ * The summernote overlay css resource reference
+ * 
+ * @author Tobias Soloschenko 
+ *
+ */
+public class SummernoteEditorOverlayCssReference extends CssResourceReference {
+
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Singleton instance of this reference
+     */
+    private static final SummernoteEditorOverlayCssReference INSTANCE = new SummernoteEditorOverlayCssReference();
+
+    /**
+     * @return the single instance of the resource reference
+     */
+    public static SummernoteEditorOverlayCssReference instance() {
+	return INSTANCE;
```

Comment:
Do you talk about Wicket's Eclipse .jar?
Or you are willing to create a new .jar with the rules this project uses?

If it is the latter then I'm fine.
@l0rdn1kk0n prefers the current code style and as a owner of the project we should respect that.